presto-docs/src/main/sphinx/release/release-0.289.rst

```diff
@@ -0,0 +1,72 @@
+=============
+Release 0.289
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug in the hash code of -0 for double and real types when ``use-new-nan-definition`` is false :pr:`23060`.
+* Fix a bug where count queries on top of aggregations with mixed global and grouped grouping sets could return wrong results.
+* Fix a bug where like predicates would only match to a newline when there was no wildcard at the end. :pr:`23404`.
+* Fix an issue where ``array_distinct``,``array_except``, ``array_intersect``, ``array_union``,  ``set_union``, and ``set_agg`` would return an error if the input was an array of rows with null fields or array of arrays with null elements.  These functions now use full "IS DISTINCT FROM" semantics for comparison.
+* Fix equijoin over timestamp with timezone types (:pr:`23319`).
+* Fix resource group state info endpoint to return correct value when caching is disabled. :pr:`23178`.
+* Improve stddev and variance functions to always return correct results when input is constant. :pr:`23447`.
+* Improve the plan check for timestamp with timezone type to cover filter pushdown :pr:`12345`.
+* Add a new configuration property ``event.query-progress-publish-interval`` to specify the time interval at which progress events should be generated. Default is every 1 minute. :pr:`23195`.
+* Add a plan checker so that queries having timestamp with timezone type will fail in prestissimo :pr:`23200`.
+* Add a session property `enable_verbose_history_based_optimizer_runtime_stats` to track latency of HBO optimizer :pr:`23241`.
+* Add an optimization to remove cross join when one side of inputs is single row of constant values. The optimization is controlled by session property `remove_cross_join_with_constant_single_row_input` :pr:`23081`.
+* Add configuration property ``warn-on-possible-nans`` and session property ``warn_on_possible_nans`` to produce a warning on division operations or comparison operations involving double or real types. Division operations are common causes of accidental creation of nans, and the semantics of comparison operations involving nans changed considerably in the most recent Presto release.
+* Add function :func:`ip_prefix_collapse` ... :pr:`23445`.
+* Add session property `enforce_history_based_optimizer_register_timeout` to enforce the maximum time HBO query registration can take :pr:`23354 `.
+* Add support for persisting full memory cache to SSD periodically on Presto C++ worker. This can be enabled by setting `cache.velox.full-persistence-interval` to a non-zero value. :pr:`23192`.
```

Comment:
```suggestion
* Add support for persisting full memory cache to SSD periodically on Presto C++ worker. This can be enabled by setting ``cache.velox.full-persistence-interval`` to a non-zero value. :pr:`23192`
```